training/distributed_training/pytorch/model_parallel/vit/submit_trainer_vit.ipynb

```diff
@@ -0,0 +1,566 @@
+{
```

Comment:
<p>If user really specified custom input using S3, does training script know what to do?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3572/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>